src/three-components/TextureUtils.js

```diff
@@ -35,10 +40,58 @@ export default class TextureManager extends EventDispatcher {
     super();
     this.config = {...defaultConfig, ...config};
     this.renderer = renderer;
-    this.cubemapGenerator = new EquirectangularToCubemap(this.renderer);
     this.envMapGenerator = new EnvMapGenerator(this.renderer);
   }
 
+  /**
+   * The texture returned here is from a WebGLRenderCubeTarget,
+   * which is not the same as a THREE.CubeTexture, and just what
+   * the current THREE.CubeCamera uses, and has the same effect
+   * when being used as an environment map.
+   *
+   * @param {THREE.Texture} texture
+   * @return {THREE.Texture}
+   */
+  equirectangularToCubemap(texture) {
+    // Each instance of EquirectangularToCubeGenerator creates a new camera,
```

Comment:
This special handling could go away if we make upstream fixes in three's generator

Reply:
Is there an open issue against Three? Maybe link it in the comment with a note about when we can take this out?

Reply:
Just filed https://github.com/mrdoob/three.js/issues/15288